src/scriptworker/constants.py

```diff
@@ -504,9 +505,9 @@
                 {
                     "firefox": ("decision", "action", "docker-image"),
                     "thunderbird": ("decision", "action", "docker-image"),
+                    "mozillavpn": ("decision", "action", "docker-image"),
```

Comment:
@escapewindow BTW, I'm not 100% sure this is what we want. Could you please confirm.

Reply:
This is what we want. We're already pinned to a dockerhub decision image, so we should be good.
https://github.com/mozilla-mobile/mozilla-vpn-client/blob/64d6b0147c698c8baba189b107fc09fc2ee011df/.taskcluster.yml#L241